docs/content/how-to-guides/configure-reactions/configure-drasi-result-reaction/_index.md

```diff
@@ -3,8 +3,137 @@ type: "docs"
 title: "Configure a Drasi Result Reaction"
 linkTitle: "Configure a Drasi Result Reaction"
 weight: 60
-toc_hide: true
-hide_summary: true
 description: >
     Learn how to configure a Drasi Result Reaction
----
\ No newline at end of file
+---
+
+The Drasi Result Reaction allows users to retrieve the current result set, or the result set at a specified timestamp, for a particular Continuous Query. This reaction provides an endpoint accessible via HTTP GET requests.
+
+## Requirements
```

Comment:
Not sure if we should have this `Requirements` section for every reaction type?
@agentofreality thoughts?

Reply:
We should aim to make each How To page a self contained, so people dont need to jump back and forth too much. If there is some replication, i think that is OK, but we need to make sure the requirements are real requirements.